src/v2/Apps/MeetTheSpecialists/Routes/MeetTheSpecialistsIndex.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from "react"
+import { Avatar, Box, Flex, Spacer, Text } from "@artsy/palette"
+import { __internal__useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { userIsAdmin } from "v2/Utils/user"
+import { useSystemContext } from "@artsy/reaction/dist/Artsy"
+import { Media } from "v2/Utils/Responsive"
+export const MeetTheSpecialistsIndex: React.FC = () => {
+  const { user } = useSystemContext()
+  const isAdmin = userIsAdmin(user)
+
+  if (!isAdmin) {
+    return null
+  }
+
+  return (
+    <>
+      <Box mt={4}>
+        <Text as="h1" variant={["xl", "xxl"]}>
+          Meet the Specialists
+        </Text>
+        <Spacer my={5} />
+        <Text color="black60" variant={["lg", "xl"]} width={["100%", "50%"]}>
+          Whether you are seeking a specific artwork for your growing collection
+          or wish to sell, our global team of sale directors is ready to source,
+          sell, advise and research on your behalf.
+        </Text>
+      </Box>
+      <Spacer mb={120} />
+      <Flex flexDirection="column">
```

Comment:
For any kind of column stuff we should use our `GridColumns` component. it taps into all of the correct design sys 12 column grid that we've been increasingly been using on our new pages: https://palette.artsy.net/elements/gridcolumns/#gatsby-focus-wrapper 

Should be a quick fix if you've got a minute, otherwise happy to help out over here.